src/hash.cr

```diff
@@ -677,6 +746,11 @@ class Hash(K, V)
     end
   end
 
+  # Inverts keys and values.
```

Comment:
Maybe mention that in case of duplicated values the last value's key becomes the new value?